src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double getFigureArea();
```

Comment:
Consider renaming the method to 'getArea' for simplicity and better readability.